### ISSUE TITLE
Add the ability to output shell script that will perform a full build

### ISSUE THREAD

**Issue by tonyfischetti** [tonyfischetti/sake#72]
Add the ability to output shell script that will perform a full build....
for people that don't have or don't want to use sake.

We will need to add one more keyword (like `visual` or `clean`) that will do this. It needs to be mentioned in the documentation and needs to be checked for in the Sakefile so that the user doesn't accidentally create a target of the same name

**Comment by kirbyfan64** [comment]
`genscript`?

**Comment by tonyfischetti** [comment]
As the target name?

**Comment by kirbyfan64** [comment]
Yup.

**Comment by tonyfischetti** [comment]
Great choice! I'll get on it!
Do you think this'll be a helpful feature?

Also, this is the last change I have on the radar for version 1. Anything you think I should add before v1 release?

**Comment by kirbyfan64** [comment]
> Do you think this'll be a helpful feature?

Definitely! This is a thing that's pretty useful but that few build systems have.

> Also, this is the last change I have on the radar for version 1. Anything you think I should add before v1 release?

Unless you can get Sake to automatically make coffee while it's running, I can't think of anything critical.

(Side note: have you ever read [this](http://gittup.org/tup/build_system_rules_and_algorithms.pdf)? Probably far too complicated to be feasible, but still an interesting read.)
